var/spack/repos/builtin/packages/wrf/package.py

```diff
@@ -258,6 +260,28 @@ def do_configure_fixup(self):
                             )
                         ofh.write(line)
 
+        if self.spec.satisfies("@4.2 %aocc"):
+            """
+            change in naming of configure.defaults is related to WRF version
+            """
```

Comment:
Can we use a comment and not a multi-line string:
```suggestion
            # In version 4.2 the file to be patched is called configure.defaults, while 
            # in earlier versions it's configure_new.defaults
```
?

Reply:
Agree, Same is resolved